### DISCUSSION TITLE
Terminating tracee-rules on EOF of all files

### DISCUSSION THREAD

**Discussion by AsafEitani** [aquasecurity/tracee#857] [Development]
When ran with the ```--input-tracee file:/path/to/file``` flag `tracee-rules` keeps running forever after finishing processing the file.
I think adding a feature, that once every file has been processed, `tracee-rules` should terminate.
This could allow automation for logs originated from `tracee-ebpf`.

Of course this feature will not be implemented for ```--input-tracee file:stdin``` flags, as there is not EOF (only termination signals).

**Comment by itaysk** [comment]
this should be the current behavior IIRC

**Comment by AsafEitani** [comment]
#874
